.husky/pre-commit

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+yarn build && git add dist/index.js
```

Comment:
While developing, I found if I did not have this in a pre-commit hook (where husky is the community choice...) I always forgot to commit dist/index.js - so this part is a really valuable addition for contributors I think

Reply:
Yeah I think you're right, this is a good addition 👍

Reply:
It never hung for me, worked every time